crates/bevy_app/src/app.rs

```diff
@@ -1007,6 +1007,7 @@ impl App {
 }
 
 fn run_once(mut app: App) {
+    trace!("no app runner detected, defaulting to run_once");
```

Comment:
I generally think that this is about a `warn!` in severity: not setting an app runner is very unusual, especially when you can call `app.update`.

Reply:
```suggestion
    warn!("No app runner detected. defaulting to run_once. If this is intentional, consider using `App::update` directly instead.");
```

Reply:
While it is unusual, do we want a `warn!` for even the bare examples? At least with `debug!` and `trace!` visibility is usually gated behind an environment variable.

```rust
use bevy::prelude::*;
fn main() {
  App::new().run();
}
```